.ado/templates/prep-and-pack-nuget.yml

```diff
@@ -73,53 +94,97 @@ steps:
-      parameters:
-        packageId: Microsoft.ReactNative.Managed
-        packageVersion: ${{parameters.npmVersion}}
-        slices: ${{ parameters.slices }}
-        buildProperties: CommitId=${{parameters.publishCommitId}};nugetroot=${{parameters.nugetroot}};baseconfiguration=$(baseConfiguration);baseplatform=$(basePlatform)
-        codesignBinaries: ${{ parameters.signMicrosoft }}
-        codesignNuget: ${{ parameters.signMicrosoft }}
+    - ${{ if containsValue(parameters.slices.*.configuration, 'Debug') }}:
+      - template: prep-and-pack-single.yml
+        parameters:
+          outputPackage: Microsoft.ReactNative.Managed.Debug
+          nuspec: Microsoft.ReactNative.Managed
+          slices: $(debugSlices)
+          packageVersion: ${{parameters.npmVersion}}
+          codesignBinaries: ${{ parameters.signMicrosoft }}
+          codesignNuget: ${{ parameters.signMicrosoft }}
+          buildProperties: CommitId=${{parameters.publishCommitId}};nugetroot=${{parameters.nugetroot}};baseconfiguration=Debug;baseplatform=$(debugBasePlatform)
+    - ${{ if containsValue(parameters.slices.*.configuration, 'Release') }}:
+      - template: prep-and-pack-single.yml
+        parameters:
+          outputPackage: Microsoft.ReactNative.Managed
+          slices: $(releaseSlices)
+          packageVersion: ${{parameters.npmVersion}}
+          codesignBinaries: ${{ parameters.signMicrosoft }}
+          codesignNuget: ${{ parameters.signMicrosoft }}
+          buildProperties: CommitId=${{parameters.publishCommitId}};nugetroot=${{parameters.nugetroot}};baseconfiguration=Release;baseplatform=$(releaseBasePlatform)
 
   - ${{ if eq(parameters.packMicrosoftReactNativeManagedCodeGen, true) }}:
     - template: prep-and-pack-single.yml
       parameters:
-        packageId: Microsoft.ReactNative.Managed.CodeGen
+        outputPackage: Microsoft.ReactNative.Managed.CodeGen
         packageVersion: ${{parameters.npmVersion}}
         buildProperties: CommitId=${{parameters.publishCommitId}};nugetroot=${{parameters.nugetroot}};baseconfiguration=$(baseConfiguration);baseplatform=$(basePlatform)
         codesignBinaries: ${{ parameters.signMicrosoft }}
         codesignNuget: ${{ parameters.signMicrosoft }}
 
   - ${{ if eq(parameters.packMicrosoftReactNativeProjectReunion, true) }}:
-    - template: prep-and-pack-single.yml
-      parameters:
-        packageId: Microsoft.ReactNative.ProjectReunion
-        packageVersion: ${{parameters.npmVersion}}
-        buildProperties: CommitId=${{parameters.publishCommitId}};nugetroot=${{parameters.nugetroot}};baseconfiguration=$(baseConfiguration);baseplatform=$(basePlatform)
-        codesignBinaries: ${{ parameters.signMicrosoft }}
-        codesignNuget: ${{ parameters.signMicrosoft }}
+    - ${{ if containsValue(parameters.slices.*.configuration, 'Debug') }}:
+      - template: prep-and-pack-single.yml
+        parameters:
+          outputPackage: Microsoft.ReactNative.ProjectReunion.Debug
```

Comment:
@AgneLukoseviciute we currently create this package based on reunion 0.8 (we use the winui3 preview4 bits of microsoft.reactnative.dll, but then project it based on the cswinrt bits from reunion 0.8, so it's a bit of a frankenstein).
In any case, we are working on upgrading this to Microsoft.ReactNative.WinAppSDK 1.0 (note the package rename)
The ProjectReunion/WinAppSDK package contains the .net projection (and the DLL), so it doesn't have the doc strings anyway so the value of debug bits is reduced.
I'm not sure we need the debug bits for the ProjectReunion package (esp if it will be replaced by winappsdk) but happy to be convinced otherwise.

Reply:
I'm has no objects to omitting debug Reunion. This PR just added Debug packages for every package which shipped binaries built in a specific configuration.